src/assets/server/rest/v1/schemas/transaction/transactions-get.json

```diff
@@ -0,0 +1,87 @@
+{
+  "title": "Get Transactions Request",
+  "type": "object",
+  "properties": {
+    "ChargeBoxID": {
+      "$ref": "chargingstation.json#/definitions/id"
```

Comment:
Do you think we need a differentiation between id and ids here? Like it is done in common.json.
My thought is, that we have some requests where we only want to reach one station and some where we have multiple stations. 
So far I did not experience any error because of this, so not urgent I guess.

Reply:
Yes you're right. 
Added a specific parameters "ids" and enforced the "id" parameter